### ISSUE TITLE
Could it use for Walkman?

### ISSUE THREAD

**Issue by SeeenW** [XyLe-GBP/ATRACTool-Reloaded#12]
I have a Walkman NW-S744 and it can play atrac audio(even atrac advanced lossless),but i dont know how to do.

**Comment by XyLe-GBP** [comment]
Sorry for the delay in replying.
At this time, we are not looking to optimize the application for the Walkman.
This is because the application is only targeted for Sony's Playstation devices.
If you have a better idea, we can listen to your suggestion.
For example, if you have software that can convert to ATRAC for Walkman, we might be able to incorporate that in some way.

**Comment by SeeenW** [comment]
Ummmmmm........I found that Sound Forge is a professional music editor,but it can convert every types of music files to ATRAC3/3+/Lossless,and Sony's Media GO can edit ATRAC audio file's tags.

**Comment by XyLe-GBP** [comment]
Is the file format you want to use on your Walkman one of .omg .oma .aa3 .atp .hma?
ATRAC is a Sony proprietary standard, so the conversion library must necessarily be a Sony one.
It may be possible to implement the functionality in an application by using the OpenMG Lib.
If that is acceptable, I will consider implementing it, but since I am doing all the coding and debugging work myself, it will take some time to release it.

**Comment by SeeenW** [comment]
Yeah but I don't know Walkman can use .hma files.....and Sound Forge seems can convert to .aa3 only.
(PSP can read these format music files in Music too,and sounds clearer than MP3 or AAC format music).
BTW,thanks a lot.
